common/src/main/java/org/opensearch/ml/common/utils/ModelInterfaceUtils.java

```diff
@@ -0,0 +1,623 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.utils;
+
+import lombok.extern.log4j.Log4j2;
+import org.opensearch.ml.common.MLModel;
+import org.opensearch.ml.common.connector.Connector;
+import org.opensearch.ml.common.transport.register.MLRegisterModelInput;
+
+import java.util.Map;
+
+@Log4j2
+public class ModelInterfaceUtils {
+
+    private static final String GENERAL_CONVERSATIONAL_MODEL_INTERFACE_INPUT = "{\n" +
```

Comment:
Should we maintain any kind of version for Model Interface? What happens if we want to add another field to the model interface later?

Reply:
It's will be bounded by blueprints in the future. So if we want add another field we just directly introduce a new variable